### ISSUE TITLE
Image runs as root user

### ISSUE THREAD

**Issue by alex1989hu** [bitnami/bitnami-docker-sonarqube#16]
**Description**
There is no `USER` statement in any Dockerfile

**Steps to reproduce the issue:**

1. Start container e.g. `docker run -it --rm --name test bitnami/sonarqube:8.1.0-debian-10-r33 whoami`
2. Verify that `Dockerfile` [content](https://github.com/bitnami/bitnami-docker-sonarqube/blob/8.1.0-debian-10-r33/8/debian-10/Dockerfile) does not contain `USER` statement

**Describe the results you received:**
```bash
$ docker run -it --rm --name test bitnami/sonarqube:8.1.0-debian-10-r33 whoami

Welcome to the Bitnami sonarqube container
Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-sonarqube
Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-sonarqube/issues
Send us your feedback at containers@bitnami.com

root
```

**Describe the results you expected:**
Dockerfile containers `USER` statement

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
N/A
```

- Output of `docker info`:

```
N/A
```

- Output of `docker-compose version` (if applicable):

```
N/A
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
N/A

**Comment by miguelaeh** [comment]
Hi @alex1989hu ,
This image is **NOT** a non-root image, so that is the expected behavior.
Could you share with us your use case so we can evaluate if move it to non root?

**Comment by alex1989hu** [comment]
The upstream image also runs as non-root, so I was surprised when I saw this image runs as root. Therefore, sonarqube does not need to run as root. We use bitnami images due to the fact they add extra level of security by setting a non-root user. I think the common use case is that avoid root images.

**Comment by miguelaeh** [comment]
HI @alex1989hu ,
Thank you very much for shring the informatin, I will create an internal task to evaluate if we should migrate this image to non-root.
Regards.

**Comment by alex1989hu** [comment]
It is still an open issue

/reopen

**Comment by miguelaeh** [comment]
Hi @alex1989hu ,
The bot closes the issue automatically, thanks for reopen it, we already have the task to fix it.
Regards.

**Comment by alex1989hu** [comment]
@miguelaeh: could you share possible ETA? 1 month passed already

**Comment by carrodher** [comment]
Hi,

We already have several infrastructure assets migrated to the non-root approach, now we are working on the required changes in another kind of assets like web applications. We have a task included in our backlog to move SonarQube, but we also have other priorities. As the impact of the issue is not critical (in the sense that there is no security issue or a bug affecting the published solution), there are no immediate plans to work on it.

We will update this thread as soon as we have more news.

**Comment by marcosbc** [comment]
Hi @alex1989hu, we just released [`SonarQube 9.0.0-10-debian-10-r0`](https://github.com/bitnami/bitnami-docker-sonarqube#900-debian-10-r0) which includes major changes, and the source code is now available in the [`rootfs` folder](https://github.com/bitnami/bitnami-docker-sonarqube/tree/master/9/debian-10/rootfs) in this repo.

As part of these changes, the container image is non-root by default. Therefore, I'm going to close this issue as resolved.
